torchok/models/necks/segmentation/hrnet_segmentation_neck.py

```diff
@@ -11,7 +11,7 @@
 @NECKS.register_class
 class HRNetSegmentationNeck(BaseNeck):
     """HRNet neck for segmentation task. """
-    def __init__(self, in_channels: Union[List[int], Tuple[int, ...]], start_block: int = 2):
+    def __init__(self, in_channels: Union[List[int], Tuple[int, ...]], start_block: int = 1):
```

Comment:
Why we still need start_block when we have feature hooks?

Reply:
Removed